.github/workflows/conformance.yml

```diff
@@ -1,6 +1,9 @@
 name: Conformance (Main)
 
 on:
+  push:
+    branches: ["tf-conformance"]
+
```

Comment:
Guessing this was for testing and should be removed, maybe worth adding a separate GitHub label flag for `conformance/kind` vs `conformance/all` or something.

Reply:
It was for testing - will remove today.

`conformance/all` IMO implies your code is having conformance tests run against it, but that's not the case. This is because we don't build and publish images for a PR that can run on cloud providers. So really, all you can do with the cloud providers in a PR is test the setup code itself.

Reply:
We could always add something like `confromance/test-cloud-infrastructure`